packages/inferno-compat/src/SVGDOMPropertyConfig.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * borrowed React SVGDOMPropertyConfig
- * @link https://github.com/facebook/react/blob/c78464f8ea9a5b00ec80252d20a71a1482210e57/src/renderers/dom/shared/SVGDOMPropertyConfig.js
+ * @link https:github.com/facebook/react/blob/c78464f8ea9a5b00ec80252d20a71a1482210e57/src/renderers/dom/shared/SVGDOMPropertyConfig.js
```

Comment:
Yeah, there's a bug with the comment parsing code, it thinks that it's the beginning of a nested comment lol...